jedi/_compatibility.py

```diff
@@ -34,8 +34,32 @@ def close(self):
         del self.loader
 
 
-def find_module_py33(string, path=None):
-    loader = importlib.machinery.PathFinder.find_module(string, path)
+def find_module_py34(string, path=None):
+    implicit_namespace_pkg = False
+    spec = None
+    loader = None
+
+    if isinstance(path, ImplicitNSInfo):
+        old_dotted_path = path.name
+        path = path.paths
+    spec = importlib.machinery.PathFinder.find_spec(string, path)
+    if hasattr(spec, 'origin'):
+        origin = spec.origin
+        implicit_namespace_pkg = origin == 'namespace'
```

Comment:
LOL no you don't have to. I have misread this. Don't ask me how that happened. I read `implicit_namespace_pkg = origin = 'namespace'`, which just clearly didn't make any sense. Haha, sorry my bad.